### DISCUSSION TITLE
Does photoprism support generating Chinese keywords when generating search terms?

### DISCUSSION THREAD

**Discussion by 643104191** [photoprism/photoprism#3711] [Q&A] [answered]
# [demo url](https://demo-zh.photoprism.app/)

## 1. When I use `cat` to search, I found some pictures

![cat](https://image-static.segmentfault.com/318/665/3186655841-64f8444e1f4d1)

## 2. When I use "猫" to search, I can't find any pictures

![cat](https://image-static.segmentfault.com/207/556/2075562064-64f8450ab9cb5)

## 3. question

1. Does photoprism support generating Chinese keywords when generating search terms?
2. Do I have to manually change the automatically generated English keywords to Chinese in order to support searching for the desired images using Chinese keywords?

**Comment by graciousgrey** [comment, marked as answer]
Keywords are created from many different sources and can be added/edited manually. If you add Chinese keywords, you can search for them. The ones that PhotoPrism creates automatically, for example from labels or color detection, are in English and currently cannot be translated automatically. However, you can rename the labels as described here: 
https://docs.photoprism.app/user-guide/organize/labels/#rename-labels.
